azure-app-service/src/main.tf

```diff
@@ -99,7 +95,7 @@ resource "azurerm_linux_web_app" "main" {
     }
 
     application_stack {
-      docker_registry_url = local.docker_registry_url
+      docker_registry_url = "https://${var.image.registry}"
```

Comment:
`https://` was added here because every attempted deploy w/o it failed with an error saying it required a URL. I then test deployed with 2 values: `https://docker.io` and `https://massdriverwestus.azurecr.io` and using `nginx` and `massdriver/nginx` respectively, and those worked